fs-one-concepts.md

```diff
@@ -3,19 +3,45 @@
 Try your best to answer each question on your own before looking up the answer online. Once you're done writing your first answer, you can google the question and write the best answer you find.
 
 #### 1. What is Enzyme and what are some of the methods that it provides?
+Enzyme is a testing package that can be downloaded into a react app to help make testing easier.
```

Comment:
Great start! What are some of the methods Enzyme provides that make testing components easier (hint: think of looking into and interacting with components)?